test/test_helper.rb

```diff
@@ -129,7 +129,17 @@ def self.coerce(vrp)
     vrp[:configuration][:preprocessing][:partitions]&.each{ |partition| partition[:entity] = partition[:entity].to_sym } if vrp[:configuration] && vrp[:configuration][:preprocessing]
     vrp.preprocessing_partitions&.each{ |partition| partition[:entity] = partition[:entity].to_sym } if vrp.is_a?(Models::Vrp)
 
-    vrp[:relations]&.each{ |r| r[:type] = r[:type]&.to_sym }
+    vrp[:relations]&.each{ |r|
+      r[:type] = r[:type]&.to_sym
+      next if [Models::Relation::NO_LAPSE_TYPES,
+               Models::Relation::ONE_LAPSE_TYPES,
+               Models::Relation::SEVERAL_LAPSE_TYPES
+              ].any?{ |set|
+                set.include?(r[:type])
+              }
+
+      raise 'This relation does not exit in any of NO_LAPSE_RELATIONS ONE_LAPSE_RELATIONS SEVERAL_LAPSE_RELATIONS, there is a risk of incorrect management'
+    }
```

Comment:
Maybe we can do the same thing for `ON_VEHICLES_TYPES` `ON_SERVICE_TYPES` as well.

Reply:
I will write a new loop over ON_VEHICLES_TYPES and ON_SERVICE_TYPES because having a type in ON_SERVICE_TYPES but in none of [NO_LAPSE_TYPES, ONE_LAPSE_TYPES, SEVERAL_LAPSE_TYPES] still is a problem I think.

Reply:
yeah yeah, exactly, I meant a separate verification. You can repeat the same logic in a separate loop or you can raise `if [lapse_types].none?{ include} || [id_types].none?{ inlcude}`.